specs/interpolation.yml

```diff
@@ -197,6 +197,13 @@ tests:
     template: '{{#a}}{{b.c}}{{/a}}'
     expected: ''
 
+  - name: Dotted Names are not valid keys
+    desc: Dotted names shall not be parsed as valid keys
```

Comment:
@simon-ess another suggestion to avoid the impression that dotted names are never valid keys:

```suggestion
  - name: Dotted Names are never single keys
    desc: Dotted names shall not be parsed as single, atomic keys
```